packages/flutter_riverpod/lib/src/consumer.dart

```diff
@@ -4,12 +4,17 @@ import 'package:meta/meta.dart';
 import 'framework.dart';
 import 'internals.dart';
 
+// ignore: one_member_abstracts
```

Comment:
@rrousselGit I think there should exist a `ProviderReferenceBase`. Then `ProviderReference` and `WidgetReference` must extend it becase they will have a common API.